pkgs/development/python-modules/blis/default.nix

```
@@ -35,6 +35,5 @@ buildPythonPackage rec {
     homepage = "https://github.com/explosion/cython-blis";
     license = licenses.bsd3;
     platforms = platforms.x86_64;
-    maintainers = with maintainers; [ danieldk ];
```

Please leave the maintainers field empty to indicate that there is no maintainer.

```suggestion
    maintainers = with maintainers; [ ];
```

---

Please could you point me to where this policy is described? Out of the 14901 `default.nix` files that we have in the `pkgs` directory, 2922 have no maintainers attribute, whereas there are 510 with an empty maintainer list.

---

> Please could you point me to where this policy is described?

Consider it a good practice to make it clear that there is no maintainer and maybe someone picks it up more easily. Also counting occurrences is not a valid argument when we talk about best practices because many of the old packages do not pass the contributing guide.

---

But I don't really see how this communicates more clearly that there is no maintainer. If there is no `maintainers` attribute, then there is are no maintainers, right? And the absence of `meta.maintainers` is just as easy to query with Nix as whether `meta.maintainers` is an empty list.

> Also counting occurrences is not a valid argument when we talk about best practices because many of the old packages do not pass the contributing guide.

Or it is just the preferred way to indicate that a package does not have maintainers. Who knows?

Anyway, I will just merge this, because I currently have little time to maintain this set of packages. If someone wants to stick an empty maintainer list in them, power to them.

---

> If there is no maintainers attribute, then there is are no maintainers, right?

Then you say it passively but when you actively write that it is empty it is way more clear that there is no maintainer.

> And the absence of meta.maintainers is just as easy to query with Nix as whether meta.maintainers is an empty list.

My text editor can't query for missing maintainers while I fix something. 

> Anyway, I will just merge this, because I currently have little time to maintain this set of packages. If someone wants to stick an empty maintainer list in them, power to them, maybe they can also take over maintainership 😀 .

That is not a very nice way to avoide a discussion.

---

> That is not a very nice way to avoide a discussion.

Until we have a rule somewhere, this seems like one of those typical bike shedding points where one reviewer tells you to do A and another reviewer tells you to do B. IMO it is not worth holding up PRs or using people's valuable time (both of the reviewer and of the submitter) for personal preferences where there is no consensus.

Removing `maintainers` or making it an empty list, I don't care either way. If we want to have consistency, we should write this down in the manual and update the derivations that do not have a `maintainers` field. Until then, I don't think it's worth making requests either way in reviews.